public/locales/pt/home.json

```diff
@@ -1,10 +1,13 @@
 {
     "title": "checagem de afirmações",
-    "subtitle": "das suas personalidades favoritas",
+    "subtitle0": "das suas",
+    "subtitle1": "personalidades favoritas", 
     "statsPersonalities": "personalidades",
     "statsClaims": "afirmações",
     "statsClaimReviews": "afirmações revisadas",
-    "statsFooter": "contribua para uma internet livre de desinformação",
+    "statsFooter": "Contribua para uma internet livre de desinformação",
     "createAccountButton": "Crie sua conta",
-    "seeMorePersonalitiesButton": "Veja mais personalidades"
+    "seeMorePersonalitiesButton": "Veja mais personalidades",
+    "sectionTitle1": "Popular hoje",
```

Comment:
We're not implementing "Popular Today" logic. Let's change that to a simple title: "Personalidades"/"Personalities".